com.unity.cinemachine/Tests/Runtime/CmColliderTests.cs

```diff
@@ -88,37 +80,30 @@ public IEnumerator CheckDampingWhenOccluded()
             m_Collider.DampingWhenOccluded = 1;
             var originalCamPosition = m_Vcam.State.GetFinalPosition();
             
-            yield return null; 
-            m_Brain.ManualUpdate();
+            yield return UpdateCinemachine();
             
             Assert.That(originalCamPosition, Is.EqualTo(m_Vcam.State.GetFinalPosition()).Using(Vector3EqualityComparer.Instance));
 
             var obstacle = GameObject.CreatePrimitive(PrimitiveType.Cube);
             obstacle.transform.position = originalCamPosition; // place obstacle so that camera needs to move
 
-            yield return WaitForOnePhysicsFrame();
-            m_Brain.ManualUpdate();
+            yield return WaitForOnePhysicsFrame(); // ensure that moving the collider (obstacle) takes affect
+            yield return UpdateCinemachine();
             
             // we are pulling away from obstacle
             Assert.That(originalCamPosition, Is.Not.EqualTo(m_Vcam.State.GetFinalPosition()).Using(Vector3EqualityComparer.Instance));
             Assert.That(new Vector3(0,0,-4.778574f), Is.EqualTo(m_Vcam.State.GetFinalPosition()).Using(Vector3EqualityComparer.Instance));
             
             var previousPosition = m_Vcam.State.GetFinalPosition();
-            var timerStart = CinemachineCore.CurrentTime;
-            do
-            {
-                m_Brain.ManualUpdate();
-                yield return null;
-            } while ((CinemachineCore.CurrentTime - timerStart) < 0.5f);
-            m_Brain.ManualUpdate();
+            yield return WaitForSeconds(0.5f);
+            yield return UpdateCinemachine();
             
             Assert.That(previousPosition, Is.Not.EqualTo(m_Vcam.State.GetFinalPosition()).Using(Vector3EqualityComparer.Instance));
-            Assert.That(new Vector3(0,0,-4.4f), Is.EqualTo(m_Vcam.State.GetFinalPosition()).Using(Vector3EqualityComparer.Instance));
-            
+            Assert.That(new Vector3(0,0,-4.423887f), Is.EqualTo(m_Vcam.State.GetFinalPosition()).Using(Vector3EqualityComparer.Instance));
```

Comment:
I'm really not a fan of these precise numbers.  To check damping, there should be some sort of loop that checks that the thing has moved _towards_ the target every frame, but not all the way until sufficient time has passed.  There need be no precise numbers.